### ISSUE TITLE
Failure in building freedomotic-model 5.6-SNAPSHOT

### ISSUE THREAD

**Issue by ritesh-pandey** [freedomotic/freedomotic#474]
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project freedomotic-model: Compilation failure: Compilation failure: 
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[33,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomPoint.java:[23,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomPoint.java:[29,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomShape.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomShape.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomShape.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomShape.java:[32,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomShape.java:[33,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Behavior.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Behavior.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Behavior.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Behavior.java:[32,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Behavior.java:[33,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/PropertiesBehavior.java:[23,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/PropertiesBehavior.java:[29,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[28,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[29,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[30,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[36,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[37,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Zone.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Zone.java:[28,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Zone.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomPolygon.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomPolygon.java:[30,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[30,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[31,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[32,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[33,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[39,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[40,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[41,2] cannot find symbol
[ERROR]   symbol: class XmlType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/MultiselectionListBehavior.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/MultiselectionListBehavior.java:[30,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/marketplace/IPluginPackage.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/marketplace/IPluginPackage.java:[30,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Tuples.java:[28,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Tuples.java:[29,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Tuples.java:[30,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Tuples.java:[38,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Tuples.java:[39,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Config.java:[34,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Config.java:[35,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Config.java:[36,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Config.java:[42,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/DataBehavior.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/DataBehavior.java:[33,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/marketplace/IMarketPlace.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/marketplace/IMarketPlace.java:[30,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/marketplace/IPluginCategory.java:[24,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/marketplace/IPluginCategory.java:[30,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/BooleanBehavior.java:[22,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/BooleanBehavior.java:[28,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomEllipse.java:[23,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomEllipse.java:[29,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Environment.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Environment.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Environment.java:[28,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Environment.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Environment.java:[35,2] cannot find symbol
[ERROR]   symbol: class XmlAccessorType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/RangedIntBehavior.java:[22,33] package javax.xml.bind.annotation does not exist
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/RangedIntBehavior.java:[28,2] cannot find symbol
[ERROR]   symbol: class XmlRootElement
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/geometry/FreedomShape.java:[33,18] cannot find symbol
[ERROR]   symbol: variable XmlAccessType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[217,6] cannot find symbol
[ERROR]   symbol:   class XmlTransient
[ERROR]   location: class com.freedomotic.model.object.Representation
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Representation.java:[226,6] cannot find symbol
[ERROR]   symbol:   class XmlTransient
[ERROR]   location: class com.freedomotic.model.object.Representation
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Behavior.java:[33,18] cannot find symbol
[ERROR]   symbol: variable XmlAccessType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[37,18] cannot find symbol
[ERROR]   symbol: variable XmlAccessType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/Properties.java:[41,6] cannot find symbol
[ERROR]   symbol:   class XmlElement
[ERROR]   location: class com.freedomotic.model.object.Properties
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/object/EnvObject.java:[40,18] cannot find symbol
[ERROR]   symbol: variable XmlAccessType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Zone.java:[43,6] cannot find symbol
[ERROR]   symbol:   class XmlTransient
[ERROR]   location: class com.freedomotic.model.environment.Zone
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Zone.java:[160,6] cannot find symbol
[ERROR]   symbol:   class XmlTransient
[ERROR]   location: class com.freedomotic.model.environment.Zone
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Tuples.java:[39,18] cannot find symbol
[ERROR]   symbol: variable XmlAccessType
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Config.java:[48,6] cannot find symbol
[ERROR]   symbol:   class XmlElement
[ERROR]   location: class com.freedomotic.model.ds.Config
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/ds/Config.java:[65,6] cannot find symbol
[ERROR]   symbol:   class XmlTransient
[ERROR]   location: class com.freedomotic.model.ds.Config
[ERROR] /home/ritesh/github/freedomotic/framework/freedomotic-model/src/main/java/com/freedomotic/model/environment/Environment.java:[35,18] cannot find symbol
[ERROR]   symbol: variable XmlAccessType
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :freedomotic-model
```

OS
```
Distributor ID:	Ubuntu
Description:	Ubuntu 19.04
Release:	19.04
Codename:	disco
```
Kernel
```
Linux truebil 5.0.0-30-generic #32-Ubuntu SMP Wed Sep 18 00:21:33 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```
Java - 
```
openjdk 11.0.4 2019-07-16
OpenJDK Runtime Environment (build 11.0.4+11-post-Ubuntu-1ubuntu219.04)
OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Ubuntu-1ubuntu219.04, mixed mode, sharing)
```

**Comment by amenak77** [comment]
Hi ritesh-pandey,
it could be a maven dependency problem; you can have some suggestions here  https://stackoverflow.com/questions/42525139/maven-build-compilation-error-failed-to-execute-goal-org-apache-maven-plugins  Best regards, Alberto

**Comment by mcicolella** [comment]
@ritesh-pandey 
The problem was related to JAXB removed from Java SDK on JEP 320 (http://openjdk.java.net/jeps/320).
Fixed in b1221af6d14a5178710100346a0e48002bc49287
